cassandra/1.cql

```diff
@@ -12,7 +12,8 @@ CREATE TABLE devgym.task (
   year TIMESTAMP,
   type TEXT,
   timeuuid TIMEUUID,
-  task_description TEXT,
+  name TEXT,
```

Comment:
Not really. We need display list of tasks on index page. They should have concise names. I do not think users will have difficulties to set task names. Most of algorithms are already have well defined and popular names. So users will pick up them for sure